Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.cs

```diff
@@ -179,6 +190,21 @@ public class SlidableListItem : ContentControl
         private AnimationSet _leftCommandAnimationSet;
         private AnimationSet _rightCommandAnimationSet;
 
+        private ControlTemplate _previousTemplateUsed;
+        private object _swipeControl;
+        private object _leftSwipeItems;
+        private object _leftSwipeItem;
+        private object _rightSwipeItems;
+        private object _rightSwipeItem;
+
+        private bool UsingSwipeControl => UseSwipeControlWhenPossible && IsSwipeControlSupported;
+
+        /// <summary>
+        /// Gets a value indicating whether <see cref="SwipeControl"/> is supported
+        /// </summary>
+        public static bool IsSwipeControlSupported => (bool)(_isSwipeControlSupported ??
```

Comment:
You can use property initialization instead of doing all that :)

Reply:
Done